server/package.json

```diff
@@ -58,7 +58,6 @@
     "@types/sinon": "10.0.6",
     "c3-linearization": "0.3.0",
     "chai": "4.3.4",
-    "codecov": "3.8.3",
```

Comment:
The `codecov` package is no longer needed as upload is through the github action.